how-to-deploy.md

```diff
@@ -0,0 +1,106 @@
+# xcube-geodb-openeo deployment
```

Comment:
I think this whole how-to-deploy should not be part of the public repo, as it contains very specific internal information. Please move to https://github.com/bcdev/service-eurodatacube/tree/master/howtos and create a new directory for how_to_xcube_geodb_openeo

Reply:
Done and made a respective PR!